common/model/card.js

```diff
@@ -43,6 +43,6 @@ export function convertItemToCardProps(
           },
         },
       },
-    link: item.promo && item.promo.link,
+    link: (item.promo && item.promo.link) || `/pages/${item.id}`,
```

Comment:
Don't we want to use link resolvers for this?

Reply:
quite possibly – I'll have to refamiliarise myself with how they work

Reply:
…oh right, that looks quite straightforward. Yes, you're right, we definitely should be using them. Thanks.